pkg/api/handlers/compat/images.go

```diff
@@ -408,7 +409,7 @@ func imageDataToImageInspect(ctx context.Context, l *libimage.Image) (*handlers.
 		Variant:         "",
 		VirtualSize:     info.VirtualSize,
 	}
-	return &handlers.ImageInspect{ImageInspect: dockerImageInspect}, nil
+	return &handlers.ImageInspect{Container: "", ImageInspect: dockerImageInspect}, nil
```

Comment:
That's the default value for a string field, so unless there's a reason to be explicitly setting it that I'm missing, it can be omitted.